src/vm/methodtable.h

```diff
@@ -4141,7 +4143,7 @@ public :
     // if enum_flag_enum_flag_HasIndirectParent is set. The indirection is offset by offsetof(MethodTable, m_pParentMethodTable).
```

Comment:
The indirection for parent method table is setup in special way as described by this comment. It is done so that the casting helpers that walk the inheritance chain does not have to check whether there is indirection or not. This special setup got broken by this change.